cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/Functions.scala

```diff
@@ -132,6 +133,7 @@ object TimeUnit extends Enumeration {
   val Week = Value("week")
   val Month = Value("month")
   val Year = Value("year")
+  val Date = Value("date")
```

Comment:
If you want to add a `Date` function for MySQL, that is totally fine. but it is not a `unit` for sure.